Adding API AM production warnings to relevant pages (#2849)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/protect-your-api/before-you-begin/index.md

---
title: Before you begin
---

> > **Note**: [Okta’s Developer Edition](/signup/) makes most key developer features available by default for testing purposes. Please note that [API Access Management](/docs/concepts/api-access-management/) — a requirement for protecting API endpoints — is an optional add-on in production.

Reviewer: ```suggestion
> **Note**: [Okta’s Developer Edition](/signup/) makes most key developer features available by default for testing purposes. [API Access Management](/docs/concepts/api-access-management/) — a requirement for protecting API endpoints — is an optional add-on for production environments.
```
